pkg/parser/ansible/ini/config/parser.go

```diff
@@ -13,7 +13,7 @@ import (
 type Parser struct {
 }
 
-func (p *Parser) Resolve(fileContent []byte, filename string) ([]byte, error) {
+func (p *Parser) Resolve(fileContent []byte, filename string, _ bool) ([]byte, error) {
```

Comment:
since you are poking around this file, maybe swap filename for "_" since it also not being used 😄